atlas/templates/Makefile.common.gotmpl

```diff
@@ -90,33 +90,33 @@ push: .push-$(IMAGE_NAME)-$(IMAGE_VERSION)
 
 .PHONY protobuf: protobuf-atlas
 protobuf-atlas:
-	@$(GENERATOR) \
+	$(GENERATOR) \
 	$(PROTOBUF_ARGS) \
 	$(PROJECT_ROOT)/pkg/pb/service.proto
 
 .PHONY vendor: vendor-atlas
 vendor-atlas:
-	@go mod tidy
-	@go mod vendor
-	@go mod download
+	go mod tidy
+	go mod vendor
+	go mod download
```

Comment:
I don't think there's any value to `go mod download` after `go mod tidy`/`vendor`. It will already have downloaded all the modules.